ngraph/core/reference/src/runtime/reference/fft.cpp

```diff
@@ -482,6 +482,19 @@ namespace ngraph
 
                     return result;
                 }
+
+                std::vector<int64_t> lengths_except_given_axis(const std::vector<int64_t>& v,
+                                                               int64_t axis)
```

Comment:
Do we have real cases with negative axis?

Reply:
No.

Reply:
In this case why do we need to support negative axis?
@lazarevevgeny What do you think? Do we need to support negative axis in the future?

Reply:
I think that we should not limit ourselves. So, lets keep current implementation.